flight/targets/omnibusf3/bl/pios_board.c

```diff
@@ -0,0 +1,88 @@
+/**
+ ******************************************************************************
+ * @addtogroup Bootloader Bootloaders
+ * @{
+ * @addtogroup OMNIBUSF3 OmnibusF3 and clones
```

Comment:
Can you make this just OmnibusF3, otherwise it's a bit too long in the menu tree IMO. Can put further explanation in @brief

Reply:
OK, will look through my briefs